metadata.json

```diff
@@ -55,7 +55,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 4.10.0 < 7.0.0"
+      "version_requirement": ">= 5.5.8 < 7.0.0"
```

Comment:
`README.md` needs to be updated accordingly.

Reply:
fixed